gradle.properties

```diff
@@ -1,2 +1,2 @@
-org.gradle.jvmargs=-Xmx2g
+org.gradle.jvmargs=-Xmx4g
```

Comment:
Out of curiosity, why was this change made in this PR? (I wouldn't be surprised if making this change is fitting for the new MRC, but I'm not sure how this change relates to removing LiveWindow)

Reply:
Accidentally committed because I had to make the change locally to successfully build with Gradle.  Given we no longer have Win32 builds I think this is safe to leave?

Reply:
Yeah, this change is probably fine (especially since it's for 2027 with new MRC, though apparently the [RFP](https://www.firstinspires.org/sites/default/files/uploads/resource_library/frc/frc-ftc-RFP-2027-robot-controller.pdf) only requires 3GB of RAM available to user code)- I was just wondering if there was a connection to removing LW that I was missing.

Reply:
The MRC has 4 gb of RAM. https://community.firstinspires.org/introducing-the-future-mobile-robot-controller

But that setting is irrelevant for the MRC or the roborio, it's for the computer building allwpilib. Even with on robot development, I don't think that extends to building allwpilib.

Reply:
Yes, the entire MRC has 4 GB of RAM, but per MRC20 in the [RFP](https://www.firstinspires.org/sites/default/files/uploads/resource_library/frc/frc-ftc-RFP-2027-robot-controller.pdf), only 3 GB needs to be available to user code (since other programs running on the MRC would take up some memory).

Regardless, you're right that the settings are only for building allwpilib with Gradle, which would not be done on the MRC, so this discussion is moot. (The conservation is good to be resolved)